samtranslator/internal/schema_source/aws_serverless_api.py

```diff
@@ -173,6 +173,7 @@ class EndpointConfiguration(BaseModel):
 TracingEnabled = Optional[PassThroughProp]
 OpenApiVersion = Optional[Union[float, str]]  # TODO: float doesn't exist in documentation
 AlwaysDeploy = Optional[bool]
+IndividualRecordSet = Optional[bool]
```

Comment:
Would under `Domain` or `Route53` make more sense? Or does it make sense even when they aren't set?

Reply:
I think under `Route53` makes more sense.  Make it unset by default is for backward compatibility